react/react1/week1/todo-list/src/componate/TodoList.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+function TodoList(props) {
+    return <div>
+        <ul>
+            <li>{props.description}</li>
+            <li>{props.date}</li>
```

Comment:
the todo's date isn't a todo item, so it shouldn't be a list item

Reply:
As I can see in output in assignment show the date with description.

Reply:
yeah, but as you can see, the date is part of the todo item, you shouldn't create a list item for the dates.
```
Todo List

* Get out of bed, Wed Sep 13 2017
* Brush teeth, Thu Sep 14 2017
* Eat breakfast, Fri Sep 15 2017
```